packages/jsii-pacmak/lib/targets/dotnet/dotnetgenerator.ts

```diff
@@ -676,11 +676,7 @@ export class DotNetGenerator extends Generator {
       return;
     }
 
-    this.code.openBlock(
```

Comment:
I removed this because of #3710, which I imagine is a better way of doing what this does, but for all languages.

Reply:
No we need a runtime opt-out to exist, too.